Framework/Backend/services/notification.js

```diff
@@ -65,23 +65,30 @@ class NotificationService {
 
   /**
    * Sends a message to selected topic
-   * @param {string} topic Kafka topic
-   * @param {string} message message to be sent
+   * @param {string} tag Bookkeeping tag
```

Comment:
Is the tag `string` or `array` or can it be both?
I can see it in the parameters as string but the code checks if  it is an array

Reply:
yes, it can be both, I'll correct jsdoc